lms/services/digest.py

```diff
@@ -138,6 +151,37 @@ def instructor_digest(self, h_userid):
                 # The user isn't an instructor in this course.
                 continue
 
+            course_assignments = []
+
+            for assignment_info in self.assignment_infos:
```

Comment:
Discussed in Slack but I don't think this is possible because we don't want to do one SQL query per course for a potentially large number of courses, we want one big query to get all the assignments for all the annotations/courses, that way the total number of queries per batch of emails is constant.